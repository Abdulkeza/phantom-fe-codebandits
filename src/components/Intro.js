import React from 'react';
import Button from './Button.js';
import bus from '../assets/bus.png';

const Intro = () => {
  return (
    <div className="flex flex-col xl:flex-row">
      <div>
        <h1 className="flex flex-col text-5xl font-medium mx-28 mt-24">
          <span>Track bus location,</span>
          <span>Save your time.</span>
        </h1>
        <p className="flex flex-col mx-40 mt-8">
          <span>Check the location of the buses</span>
          <span>on their respective routes</span>
        </p>
        <Button
          name="Start"
          styles="bg-blue-600 text-white rounded-3xl mt-10 ml-6 px-12 py-2 ml-48"
        />
      </div>
      <div className="w-96 ml-auto mr-48 mt-28">
        <img src={bus} alt="Bus" />
      </div>
    </div>
  );
};

export default Intro;
