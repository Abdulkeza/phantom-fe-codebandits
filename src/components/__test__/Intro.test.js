import React from 'react';
import renderer from 'react-test-renderer';
import Intro from '../Intro.js';

describe('Intro', () => {
  it('should render Intro', () => {
    const intro = renderer.create(<Intro />).toJSON();
    expect(intro).toMatchSnapshot();
  });
});
