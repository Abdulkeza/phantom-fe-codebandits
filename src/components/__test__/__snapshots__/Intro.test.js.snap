// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro should render Intro 1`] = `
<div
  className="flex flex-col items-center lg:flex-row xl:flex-row"
>
  <div>
    <h1
      className="flex flex-col text-3xl md:text-5xl xl:text-5xl font-raleway font-extrabold mx-12 xl:mx-28 mt-12 xl:mt-32"
    >
      <span>
        Track bus location,
      </span>
      <span
        className="ml-4 md:ml-10"
      >
        Save your time.
      </span>
    </h1>
    <p
      className="flex flex-col text-sm md:text-xl xl:text-base mx-20 md:ml-40 xl:ml-52 mt-10"
    >
      <span>
        Check the location of the buses
      </span>
      <span
        className="ml-4"
      >
        on their respective routes
      </span>
    </p>
    <div>
      <button
        className="bg-primary hover:bg-hover text-white rounded-3xl mt-12 px-12 xl:px-12 py-2 ml-28 md:ml-52 xl:ml-64 rounded-xl py-3 px-5"
        type="button"
      >
        Start
      </button>
    </div>
  </div>
  <div
    className="w-full px-20 lg:w-80 xl:w-96 lg:ml-auto xl:ml-auto lg:mr-36 xl:mr-36 lg:px-0 xl:px-0 mt-28"
  >
    <img
      alt="Bus"
      src=""
    />
  </div>
</div>
`;
